home.asm

```diff
@@ -1035,13 +1035,8 @@ IsHM:: ; 34df
 	ret
 ; 34e7
 
-IsHMMove:: ; 34e7
-	ld hl, HMMoves
-	ld de, 1
-	jp IsInArray
 
-
-INCLUDE "data/hm_moves.asm"
+INCLUDE "home/hm_moves.asm"
```

Comment:
I think the surrounding functions should be home/moves.asm and HMMoves should be put back into data/. Ignore if this is already done.

Reply:
There was disagreement over whether the table should go in data/ because it's data, or home/ because it's in bank 0. Just let me know which it should be in; or just merge and move it where you prefer. I just want it to be in a file *some*where, not buried in home.asm, because it's an important table for users to edit.

Reply:
In the meantime I've put the surrounding functions in their own home/ file.